### PR TITLE
Feature/refactor structure

### PR DESCRIPTION
**Pull request by SoundSonic1**
Added additional sub packages for better structure

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -32,4 +32,5 @@ jobs:
     - run: flutter upgrade
     - run: flutter clean
     - run: flutter pub get
+    - run: flutter analyze
```
Comment (omartoutounji) [inline .github/workflows/main.yml:35]: @SoundSonic1 what does flutter analyze do and why is it needed in CI?

File: lib/routes/hwyd_routes.dart [hunk 2]
```diff
@@ -0,0 +1,6 @@
+import 'package:flutter/cupertino.dart';
```
Comment (omartoutounji) [inline lib/routes/hwyd_routes.dart:1]: @SoundSonic1 Nice!

File: lib/screens/hwyd.dart [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+  _disableOnboard() async {
+    SharedPreferences prefs = await SharedPreferences.getInstance();
+    prefs.setString(_onBoardKey, json.encode(false));
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return showOnboard == true ? _buildIntroductionScreen() : JournalPage();
+  }
```
Comment (omartoutounji) [inline lib/screens/hwyd.dart:41]: @SoundSonic1 never knew you could do this. This is genius!

**Review by omartoutounji — COMMENTED** [review]
Great stuff and flutter analyze will really help when CI complains

File: .github/workflows/main.yml (hunk 1 above)
Comment (SoundSonic1) [reply .github/workflows/main.yml:35]: It does static code analysis. It complains for example if variables are unused

Comment (omartoutounji) [reply .github/workflows/main.yml:35]: Cool, yeah that’ll be useful to know if CI fails or complains

**Pull request merged**